atomistics/workflows/elastic/workflow.py

```diff
@@ -110,6 +110,7 @@ def get_C_eigval(self):
     BV=ElasticProperties.get_BV,
     BR=ElasticProperties.get_BR,
     BH=ElasticProperties.get_BH,
+    bulkmodul_eq=ElasticProperties.get_BH,
```

Comment:
This is only the case if the equilibrium structure was used as an input for the elastic matrix calculation:
```suggestion
```